test/unit/ObjectMakrTests/proliferate.js

```diff
@@ -0,0 +1,51 @@
+define(["mocks"], function (mocks) {
+    return function () {
+        var expect = require("chai").expect;
+
+        it("copies into the recipient object", function () {
+            // Arrange
+            var objectMaker = mocks.mockObjectMakr();
+
+            var obj1 = {},
+                obj2 = mocks.mockObjectProperties();
+
+            // Act
+            objectMaker.proliferate(obj1, obj2);                
+
+            // Assert
+            expect(obj1).to.deep.equal(obj2);
+        });
+
+        it("leaves properties already in the recipient unchanged", function () {
+            // Arrange
+            var objectMaker = mocks.mockObjectMakr();
+
+            var obj1 = {
```

Comment:
Prefer one variable per declaration (no commas). The code style might not have been updated in all the FSS repos, but that's the new way.